fastly-compute/geoip/.secret.json

```diff
@@ -0,0 +1,3 @@
+{
+ "token" : "1wps9CyJtrdeoDCYW6uUtXmEZNckMsdO"
+}
```

Comment:
Is this a really secret token? If so, we need to remove it (and probably add the path to .gitignore ) and rotate the secret since it's now been seen

Reply:
This was a real secret which was revoked. I audited the logs in fastly for usage and it was not abused.